CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## 1.5.0
+* Feature - Automatically re-create CloudWatch log groups and log streams after they are deleted(#95)
+* Feature - Add default fallback log group and stream names (#99)
+* Feature - Add support for special variables in log stream and group names (#108)
```

Comment:
Lets specifically say: ECS Metadata and UUID via special variables

Reply:
Users will look to the changelog to know what we added

Reply:
Got it. I will make it more specific